### DISCUSSION TITLE
Lottery Iota is SCAM

### DISCUSSION THREAD

**Discussion by Adiac21** [iotaledger/firefly#8026] [Help & Support]
Lottery Iota is SCAM I lost my funds

**Comment by Adiac21** [comment]
Check this....
![Liota](https://github.com/iotaledger/firefly/assets/87958840/a94a4915-6eb9-4505-9e55-71134946a116)

**Comment by Adiac21** [comment]
Iota me ha decepcionado soy de Nicaragua me costo mucho reunir el monto perdido.

**Comment by no8ody** [comment]
Unfortunately, this is your own fault. Such scams can happen on any DLT that charges very low fees for sending NFTS, coins or whatever.

Unfortunately, you're out of luck and hopefully you'll be more careful next time. Especially as you should never just enter your seed anywhere. IOTA can do absolutely nothing for your actions in this case and it is 100% your own fault.

**Comment by Adiac21** [comment]
caso cerrado. que mal IOTA.
